ios/README.md

```diff
@@ -110,6 +110,15 @@ Loads a specific URL which may identify a conference to join. If the specified
 URL is `nil` and the Welcome page is enabled, the Welcome page is displayed
 instead.
 
+#### setDropboxAppKey:NSString
+
+```objc
+[JitsiMeetView setDropboxAppKey:@"Your app key"];
+```
+Sets your dropbox app key.
+
+NOTE: Must be set (if at all) before `application:didFinishLaunchingWithOptions: ` for it to take effect.
```

Comment:
before that function **returns**

Reply:
Actually this is outdated now and needs to be removed. Thanks for note.